tests/load/load_support/playwright_stats.py

```
@@ -0,0 +1,228 @@
+        yield
+        await collector.checkpoint()
+        _fire_request_event(
+            user,
+            request_type=request_type,
+            name=name,
+            start_time=start_time,
+            start_perf_counter=start_perf_counter,
+            response_length=collector.finish_window(window_id),
+            exception=None,
+        )
+    except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)